kubernetes/helm/activitiquerychart/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Activiti 7 Query Service on Kubernetes (see https://github.com/Activiti/)
+name: activitiquerychart
```

Comment:
Is that possible to have a separator or to use camel case for the chart name (activiti-query-chart, activiti_query_chart, activitiQueryChart)? Or is it a convention to use only lower case with no separators ? Same question for the folder name `activitiquerychart `.

Reply:
yeah we should use hyphens as that is what most do https://github.com/kubernetes/charts/tree/master/stable

Reply:
I think this is something to discuss separately as it's not part of this change and we're not currently using these more granular charts (which is the bigger question).